File: ejercicicos con jose cuellar/Python/ejercicio14.py
#Se tienen un cultivo de café, de los cuales con el uso de la tecnología se recolectan los siguientes datos para análisis de suelo:
#PH (Acidez del suelo) rango adecuado entre 5 y 5.5.
#Materia orgánica mayor a 8
#Fosforo (P) mayor a 30
#Calcio (Ca) mayor a 3
#Magnesio (Mg) mayor a 0.9
#Potasio (K) mayor a 0.4
#Aluminio (Al) menor a 1
#Acorde a los datos recolectados identificar si el suelo es apto para la siembra cafe

ph = float(input("Ingrese el ph del suelo: "))
materia = float(input("Ingrese la materia orgánica: "))
fosforo = float(input("Ingrese el fosforo: "))
calcio = float(input("Ingrese el calcio: "))
magnesio = float(input("Ingrese el magnesio: "))
potasio = float(input("Ingrese el potasio: "))
aluminio = float(input("Ingrese el aluminio"))

if ph > 

File: ejercicicos con jose cuellar/js/js/ejercicio14.js
function calcular(){

    let ph, materia,fosforo, calcio, magnesio, potasio, aluminio

    ph = document.getElementById("ph").value 
    materia = document.getElementById("materia").value 
    fosforo = document.getElementById("fosforo").value 
    calcio = document.getElementById("calcio").value 
    magnesio = document.getElementById("magnesio").value 
    potasio = document.getElementById("potasio").value
    aluminio = document.getElementById("aluminio").value 

    print = document.getElementById("respuesta")

    if(ph > 5 && ph <=  5.5 && materia > 8 && fosforo > 30 && calcio > 3 && magnesio > 0.9 && potasio > 0.4 && aluminio < 1){
        print.innerHTML = "El suelo es apto para la siembra de café"
    }else{
        print.innerHTML = "El suelo no es apto para la siembra de café"
    }
}